app/components/DepositWithdraw/DepositWithdrawAssetSelector.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { connect } from "alt-react";
+import Translate from "react-translate-component";
+import BindToChainState from "../Utility/BindToChainState";
+import { Apis } from "bitsharesjs-ws";
+import { settingsAPIs, rudexAPIs } from "api/apiConfig";
+import GatewayStore from "stores/GatewayStore";
+import GatewayActions from "actions/GatewayActions";
+import TypeAhead from "../Utility/TypeAhead";
+
+class DepositWithdrawAssetSelector  extends React.Component {
+    render(){
+        const { props } = this;
+
+        let getCoinOption = (item)=>{
```

Comment:
Changes this to the following to find who our backer is, as well as returning the label as the backed coin instead of the backers asset.
```
let getCoinOption = (item) => {
            /* Gateway Specific Settings */
            let gateway;
            let backedCoin;

            if(item.intermediateAccount && (item.intermediateAccount == "openledger-dex" || item.intermediateAccount == "openledger-wallet")) {
                gateway = "OPEN";
                backedCoin = item.backingCoinType;
            } else if(item.gatewayWallet && (item.gatewayWallet == "rudex" || item.gatewayWallet == "rudex-gateway")) {
                gateway = "RUDEX";
                backedCoin = item.backingCoin;
            } else {
                console.log("Not Found");
                console.log(item);
            }
            
            return { id: item.symbol, label: backedCoin, gateway: gateway };
        };
```